src/schema/firefox-schemas-import.js

```diff
@@ -552,12 +555,19 @@ inner.normalizeSchema = (schemas, file) => {
       if (typeof $import !== 'string') {
         return {};
       }
+
       // menus.json and browser_action.json contain the definition of two API
       // namespaces, one just import the other one defined in the same file
       // (e.g. "contextMenus" imports "menus", "browser_action" imports "action"),
       // we want to expand those definitions otherwise they will be converted into
       // a $merge form, which is only used in the manifest validation and so the
       // eslint rules that look for the API method definitions will not work as expected.
+      const obj = apiNamespaceSchemas.find((schema) => schema.namespace === $import);
+
+      if (obj === undefined) {
+        return {};
+      }
+
```

Comment:
We need to be careful here. action API is MV3 and browser_action is MV2 and there will be differences between them.

Is the linter checking for MV2 and MV3?

Reply:
Both MV2 and MV3. It splits it properly, I'll make a comment when I push the newer fixed version.